common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -6106,6 +6107,17 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(4)
 
+  val CLOUD_DISK_MOUNTED: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.readLocalShuffleFile.cloudDiskMounted")
```

Comment:
How about renaming it to "celeborn.client.readLocalShuffleFile.sharedDisk".

There are multiple approaches to implementing shared disks, cloud disk is one of them.